dataactcore/scripts/initialize.py

```diff
@@ -136,6 +144,8 @@ def main():
                         action='store_true')
     parser.add_argument('-tempd', '--update_domain_temp', help='only update domain values not cfda',
                         action='store_true')
+    parser.add_argument('-obj', '--update_object', help='load object class to database',
```

Comment:
Arguments should have object class,  both `update_object` and `obj`, since`object` is programming concept and has a different meaning from `object_class`